sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py

```diff
@@ -13,6 +13,7 @@
 class TextAnalyticsApiVersion(str, Enum):
     """Text Analytics API versions supported by this package"""
 
+    V3_1_PREVIEW_4 = "v3.1-preview.4"
```

Comment:
after talks with the team, looks like preview 4 is completely up, so we can just make `V3_1_PREVIEW` point to v3.1-preview.4 (i think this requires one update in the readme.md as well). This should also help clean up some code in the tests, don't need the `client_kwargs` any more

Reply:
I think this will cause failures in tests for Healthcare at the moment, because the relations structure has changed in v3.1-preview.4.  I was planning to make this change before merging the feature branch into master, but for this PR I think it will cause issues.